lib/money/money.rb

```diff
@@ -84,7 +84,9 @@ def -(other)
   end
 
   def *(numeric)
-    raise TypeError, "#{numeric} is not Numeric" unless numeric.is_a?(Numeric)
+    unless numeric.is_a?(Numeric)
+      Money.deprecate("Multiplying Money with #{numeric.class.name} is deprecated and will be removed in the next major release.")
```

Comment:
Before it was impossible to multiply with a Numeric, now it will be possible + a warning? This seems to achieve the opposite.